src/Features/Core/Portable/Completion/CompletionList.cs

```diff
@@ -21,7 +22,8 @@ public sealed class CompletionList
         /// <summary>
         /// The completion items to present to the user.
         /// </summary>
-        [Obsolete("Use ItemsList instead")]
+        [Obsolete($"This property is obsolete. Use {nameof(ItemsList)} instead", true)]
```

Comment:
See, this is why I asked. I was expecting this to be `error: false`.

Reply:
hmm, it seems we have been very inconsistent in terms of using warning vs. error in IDE layer (also very inconsistent in terms of the explicitness)